### DISCUSSION TITLE
Xanthophyllum

### DISCUSSION THREAD

**Discussion by john30** [john30/ebusd#1107] [Announcements]
This is the 15th release of ebusd 3 and the 24th (="X", therefor [Xanthophyllum](https://en.wikipedia.org/wiki/Xanthophyllum)) since ebusd was completely reworked.

The major new features are a reqorked eBUS protocol engine, time fields for MQTT integration, removed dependency on libargp, as well as a bunch of bug fixes.

See [ChangeLog](https://github.com/john30/ebusd/blob/master/ChangeLog.md) and [Wiki](https://github.com/john30/ebusd/wiki) for details.

Debian 12 (Bookworm), 11 (Bullseye), and 10 (Buster) install packages for amd64, 386, armv7 (Raspberry Pi 2/3/4), and arm64 (Raspberry Pi 3/4 64 bit) are attached and will also be made available soon in the [Debian repository as described here](https://github.com/john30/ebusd-debian/blob/master/README.md).

For convenience, an armv6l (Raspberry Pi 1 and Zero/Zero W) Buster image is attached here as well (not part of the Debian repository though).

Variants of each binary with MQTT support have an additional "mqtt1" (for libmosquitto1) suffix in the name, whereas those without don't.

For Arch Linux, simply run `makepkg` in the `contrib/archlinux/ebusd/` directory.

For Alpine Linux, run `apk --repository=http://dl-cdn.alpinelinux.org/alpine/edge/community add ebusd`.

The latest release is also available as [multi-architecture Docker image on the hub](https://hub.docker.com/r/john30/ebusd) with tag ebusd:latest.

<hr /><em>This discussion was created from the release <a href='https://github.com/john30/ebusd/releases/tag/23.3'>Xanthophyllum</a>.</em>

**Comment by samm-git** [comment]
Thank you! [FreeBSD port](https://www.freshports.org/comms/ebusd/) is also updated, found a minor issue with `ebusfeed` tool (https://github.com/john30/ebusd/issues/1111). Also tested the new version with protherm hardware, seems to work as expected.

**Comment by SzosszeNET** [comment]
Sorry, but seeing "For convenience, an armv6l (Raspberry Pi 1 and Zero/Zero W) Buster image is attached here as well (not part of the Debian repository though)." but can't see an image attached, but deb files and source tars?

**Comment by apbarratt** [comment]
Hi guys, it doesn't look like 23.3 was ever released for Alpine.  Could this be done?
